domains/zendai.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "ZendaiOwl",
+        "email": "victorray91@pm.me"
+      },
+
+      "record": {
+        "CNAME": "zendaiowl.github.io"
```

Comment:
404?

Reply:
Yes cuz it's setup towards https://website.zendai.ml at the moment so not reachable on the original GH pages URL right now